xml/admin_ceph_upgrade.xml

```diff
@@ -121,8 +121,36 @@
       <para>
        Run the <command>salt-run upgrade.check</command> command to verify that
        the cluster uses the new bucket type <literal>straw2</literal> and that
-       the &adm; is not a storage node.
+       the &adm; is not a storage node. <literal>straw2</literal>  is the default
```

Comment:
Being super-pedantic: 
• 2 spaces after the straw2 literal
• I do not like a sentence beginning with an uncapitalized word. Can we change it to "The default is `straw2` for any newly-created buckets"?